### ISSUE TITLE
I18n?

### ISSUE THREAD

**Issue by ChristianUlbrich** [danielgtaylor/aglio#7]
Great piece of software and professionally looking templates. Thank you very much. But what about i18n? Do you have any plans for that? From a quick web search, one possible approach would be using i18next with JADE, as described in [http://rbeere.tumblr.com/post/41212250036/internationalization-with-express-jade-and] and the obviously easier (but IMHO dirtier) option would be to simply translate the templates into specific languages and adding location_codes (e.g. default-de.jade , default-es.jade , ...).

Either way, I might be up for some PRs if you make up your mind. :)

**Comment by Lambik** [comment]
Yes, I would want to write my API doc in many different languages, translated possibly by a professional translator. Google translate sucks.

As for the template translations: either option is fine specified by @ChristianUlbrich is fine, as right now there is none.
